README.md

```diff
@@ -1,6 +1,8 @@
 # YaST - The Squid Module #
 
-[![Travis Build](https://travis-ci.org/yast/yast-squid.svg?branch=master)](https://travis-ci.org/yast/yast-squid)
-[![Jenkins Build](http://img.shields.io/jenkins/s/https/ci.opensuse.org/yast-squid-master.svg)](https://ci.opensuse.org/view/Yast/job/yast-squid-master/)
-[![Coverage Status](https://img.shields.io/coveralls/yast/yast-squid/master.svg)](https://coveralls.io/github/yast/yast-squid?branch=master)
-
+[![Workflow Status](https://github.com/yast/yast-squid/workflows/CI/badge.svg?branch=master)](
+https://github.com/yast/yast-squid/actions?query=branch%3Amaster)
+[![Jenkins Status](https://ci.opensuse.org/buildStatus/icon?job=yast-yast-squid-master)](
+https://ci.opensuse.org/view/Yast/job/yast-yast-squid-master/)
+[![Coverage Status](https://img.shields.io/coveralls/yast/yast-squid.svg)](https://coveralls.io/r/yast/yast-squid?branch=master)
+[![inline docs](http://inch-ci.org/github/yast/yast-squid.svg?branch=master)](http://inch-ci.org/github/yast/yast-squid)
```

Comment:
Inch is not enabled for this project, https://inch-ci.org/github/yast/yast-squid fails.